File: front-end/src/app/service-models/app/v1/license/checkout-session-request-s-m.ts
import { PaymentModeSM } from "../../enums/payment-mode-s-m.enum";

export class CheckoutSessionRequestSM {
    productId!: string;
    priceId!: string;
    successUrl!: string;
    failureUrl!: string;
    paymentMode!: PaymentModeSM;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/License/CheckoutSessionRequestSM.cs
using Siffrum.Web.Payroll.ServiceModels.Enums;

namespace Siffrum.Web.Payroll.ServiceModels.v1.License
{
    public class CheckoutSessionRequestSM
    {
        public string ProductId { get; set; }
        public string PriceId { get; set; }
        public string SuccessUrl { get; set; }
        public string FailureUrl { get; set; }
        public PaymentModeSM PaymentMode { get; set; }
    }
}
